java/core/src/main/resources/rules/asana/asana.yaml

```diff
@@ -171,6 +171,7 @@ endpoints:
       - "modified_at.before"
       - "is_subtask"
       - "sort_ascending"
+      - "opt_fields"
```

Comment:
From https://github.com/Worklytics/psoxy/pull/214 (merged to main, instead of RC)